src/generic_ui/polymer/link.ts

```diff
@@ -0,0 +1,6 @@
+Polymer({
+  handleKey: function() {
+    this.click();
```

Comment:
But if that's really the way Polymer wants you to do things it's fine, I would just add some comments explaining this